src/__integ__/page-objects/async-dropdown-page.ts

```diff
@@ -25,7 +25,7 @@ declare const window: ExtendedWindow;
 export const DEBOUNCE_FILTERING_DELAY = 300;
 export const RESPONSE_PAGE_SIZE = 20;
 
-export class AsyncDropdownPage extends BasePageObject {
```

Comment:
Here and in many places below – if an export is not used anywhere in our code and not a part of our public API, remove the export.

After that, ESLint started detecting unused code which could be removed